source/tomopy/prep/stripe.py

```diff
@@ -351,19 +352,20 @@ def remove_stripe_sf(tomo, size=5, ncore=None, nchunk=None):
     return arr
 
 
-def remove_stripe_based_sorting(tomo, size=None, ncore=None, nchunk=None):
+def remove_stripe_based_sorting(tomo, size=21, dim=1, ncore=None, nchunk=None):
```

Comment:
This new default size is not equivalent to the old behavior. The old behavior is approximately:
```python
size = max(5, min(int(0.01 * tomo.shape[2]), 21)) if size is None else size
```
Is there a reason that `21` is the preferred filter size for all tomogram sizes?

Reply:
Yeap, it's not a general number. Will roll back.